integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/validation/IsDocumentTypeValidator.java

```diff
@@ -0,0 +1,22 @@
+package no.difi.meldingsutveksling.validation;
+
+import lombok.extern.slf4j.Slf4j;
+import no.difi.meldingsutveksling.ApiType;
+import no.difi.meldingsutveksling.DocumentType;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+@Slf4j
+public class IsDocumentTypeValidator implements ConstraintValidator<IsDocumentType, String> {
+
+    private ApiType api;
+
+    public void initialize(IsDocumentType constraint) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1161)